java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture6/tasks/_1/City.java

```diff
@@ -0,0 +1,40 @@
+package com.zoolatech.lecture6.tasks._1;
+import java.util.Objects;
+
+record City(String name, String country, int population) {
+
+    public String getName() {
```

Comment:
you don't need to define custom getters, toString, equals and hashCode for the record - it will be provided for you

you can only do them if you'd like to have a custom behavior